### ISSUE TITLE
Problem with generating token

### ISSUE THREAD

**Issue by Uraharadono** [shammelburg/CoreIdentity#2]
I have issues accessing methods that are requiring authorization.

Problem is that even after I have added generated **_bearer token_** (AuthController -> CreateToken) - into Auth part of either **PostMan** or **Swagger** I would get `404 error`.

I tried to specify Authorization to use bearer token like so:

`[Authorize(AuthenticationSchemes = "Bearer")]`

and specifying in **AuthenticationHelper.cs** 

```
.AddAuthentication(cfg =>
                {
                    cfg.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
                    cfg.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
                })
```

like some of the forums online suggested, but to no avail. In best case I would get `401 Unauthorized`  after changing these 2 things. 

Has anyone managed to make this work?

**Comment by shammelburg** [comment]
@Uraharadono as you are getting a 404, can you check you are calling the correct endpoint?

Also, when using Postman under Authorization of your request, make sure you select the Bearer Token from the drop down.

Can you let me know exactly what you are trying to do as well, I will then clone the project and replicate.

**Comment by Uraharadono** [comment]
@shammelburg Hey mate, as I stated, I cannot make token Authorization work ([Authorize] attribute on controller).
You can reproduce this by forking or downloading this repo, and running migrations.

After that, create user in database using AuthController -> Register (don't forget to mark it as Email confirmed as mailer is not working out of the box), and login. Copy generated token in Swagger Auth part and try to call method RoleController -> Get.
 That's when you should get 404.

If you make changes I explained above, you should get 401. 


EDIT because I didn't answer question what I want to do:

I honestly am looking just to make users who are logged in, access parts of app that are Authorized.

**Comment by shammelburg** [comment]
@Uraharadono yes you are right.

Found this:-
[Web api core returns 404 when adding Authorize attribute](https://stackoverflow.com/questions/52038054/web-api-core-returns-404-when-adding-authorize-attribute)

I've added `[Authorize(AuthenticationSchemes = "Bearer")]` to the top of the RoleController and now I get the 401.

I will update project soon.

Thank you

**Comment by Uraharadono** [comment]
**Edit:** Wow, I just figured we linked same blog post and I look like idiot now. Leaving this here, because this is lesson to learn for myself.

Hey man, if I can be of any help. While I was searching for the solution I came across this SO exchange:
https://stackoverflow.com/questions/52038054/web-api-core-returns-404-when-adding-authorize-attribute/52039496#52039496

and there I found this blog post: https://wildermuth.com/2018/04/10/Using-JwtBearer-Authentication-in-an-API-only-ASP-NET-Core-Project


But honestly it made no difference still, as I was unable to make it work :(
